electron/main.js

```diff
@@ -0,0 +1,37 @@
+const { app, BrowserWindow } = require('electron');
+const serve = require('electron-serve');
+
+const loadURL = serve({ directory: '../frontend/dist' });
```

Comment:
The path can be defined differently for different OSs. It's recommended to use `path` node package to do any cross-os operations with file path (eg `path.join()`)